src/components/policy/policy_external/src/sql_pt_representation.cc

```diff
@@ -645,6 +645,10 @@ bool SQLPTRepresentation::GatherFunctionalGroupings(
       *rpcs_tbl.user_consent_prompt = func_group.GetString(2);
     }
 
+    if (!func_group.IsNull(3)) {
```

Comment:
There are also changes required in sql_pt_ext_representation.cc

Reply:
Added in 988d4fca0c6a9825e3e2c301905c00705bbcc901